model/src/main/java/org/cloudfoundry/identity/uaa/util/UaaStringUtils.java

```diff
@@ -49,6 +49,8 @@ public final class UaaStringUtils {
 
     public static final String EMPTY_STRING = "";
 
+    public static final String DEFAULT_UAA_URL = "http://localhost:8080/uaa";
```

Comment:
Nice, that is a step in the right direction. Following this PR, the port should be defaulted to 8080 with the capability to override.